score-rce/src/test/java/io/cloudslang/content/remote/RunWindowsCommandTest.java

```diff
@@ -0,0 +1,17 @@
+package io.cloudslang.content.remote;
+
+import io.cloudslang.content.remote.action.RunWindowsCommand;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import java.io.*;
+/**
+ * Created by pasternd on 06/06/2016.
+ */
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({File.class, RunWindowsCommand.class})
+public class RunWindowsCommandTest {
```

Comment:
@dashapaster  Here will be a fun part of the class right Daria... but you didn't put anything in it yet...